tests/executables_src/testUtilities.cpp

```diff
@@ -264,3 +264,58 @@ BOOST_AUTO_TEST_CASE(testUserTypeToUserType_Boolean) {
 }
 
 /**********************************************************************************************************************/
+
+BOOST_AUTO_TEST_CASE(testUserTypeToUserType_String){
+  //Regular decimal branch
+  BOOST_TEST(ChimeraTK::userTypeToUserType<ChimeraTK::Boolean>(std::string("1")) == true);
+  BOOST_TEST(ChimeraTK::userTypeToUserType<ChimeraTK::Boolean>(std::string("2")) == true);
+  BOOST_TEST(ChimeraTK::userTypeToUserType<ChimeraTK::Boolean>(std::string("-123456789")) == true);
```

Comment:
Actually, the whole "to Boolean" part is already covered in the test above. We should not repeat it here but make a code comment where to find it.